modules/ROOT/pages/manage-proxy.adoc

```diff
@@ -42,16 +42,17 @@ Updating the HTTP proxy causes each machine in the cluster to restart all pods r
 . Run the following command, replacing the placeholder values with the following:
 +
 ----
-rtfctl apply http-proxy "http://<user>:<pass>@<10.0.0.1>:<8080>" --no-proxy "<1.1.1.1:8888,2.2.2.2:9999>"
+rtfctl apply http-proxy "http://<user>:<pass>@<10.0.0.1>:<8080>" --no-proxy "<1.1.1.1:8888,2.2.2.2:9999>" --confirm
```

Comment:
i don't think we should include `--confirm` in the default example. The reason is this is an operation which causes all your applications to restart, so we want you to confirm this is what you want to do

Reply:
Thank you! I've removed that.